integration_tests/constants/constants.go

```diff
@@ -414,6 +414,95 @@ IBHlx8AWzAuLUwHE7hqnYz3ROq+u3idJMgUJtIQrXWqq9oqOl6VP1WC6e/RLmN36
 a0u7AgG5ftl0Qx8nqht2Qdr+fIPivdR9JFweH/irVgTjgAo4Tj/0okgWdjS/qfIx
 41rK5Rk4u875N8SOyJIU+DPbh7+bGV6QwX35aoPckfz12nC1uwDJ1SuLI55qy1cx
 KER9kcWhvxgD0H/rtNnFVgbvQw==
+-----END CERTIFICATE-----`
+
+	RootCertWithSameSubjectAndSKID1 = `-----BEGIN CERTIFICATE-----
```

Comment:
Please extend the expiration period of the certificate to at least 100 years. Currently, it is set to expire on March 14, 2024. After this date, the tests will no longer be considered valid